sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuExpressions.scala

```diff
@@ -377,6 +376,19 @@ trait GpuString2TrimExpression extends String2TrimExpression with GpuExpression
       }
     }
   }
+
+  protected def doEval(
```

Comment:
looks like maybe from https://issues.apache.org/jira/browse/SPARK-33890

We don't have an issue to track this, we should file internal issue so we don't lose it